src/reference/asciidoc/file.adoc

```diff
@@ -216,6 +215,8 @@ All other sub-directories inclusions and exclusions are based on the target `Fil
 For example, the `SimplePatternFileListFilter` filters out directories by default.
 See https://docs.spring.io/spring-integration/api/org/springframework/integration/file/filters/AbstractDirectoryAwareFileListFilter.html[`AbstractDirectoryAwareFileListFilter`] and its implementations for more information.
 
+NOTE: Starting with version 5.5, the `FileInboundChannelAdapterSpec` of Java DSL has a convenient `recursive(boolean)` option to use the mentioned `RecursiveDirectoryScanner` in the target `FileReadingMessageSource` instead of default one.
```

Comment:
```suggestion
NOTE: Starting with version 5.5, the `FileInboundChannelAdapterSpec` of the Java DSL has a convenient `recursive(boolean)` option to use a `RecursiveDirectoryScanner` in the target `FileReadingMessageSource` instead of the default one.
```